Podfile

```diff
@@ -66,6 +75,11 @@ target 'Database' do
   pod 'Firebase/Database'
 end
 
+target 'Storage' do
```

Comment:
FirebaseStorageUI is a dynamic framework target, Storage is a static lib. Currently our build thing builds the dynamic framework and then swaps the executable with a lipo'd static lib so that way we don't have to bundle the framework manually.

As soon as Firebase ships as a dylib, we can just delete these static lib targets.